button/button.css

```diff
@@ -0,0 +1,65 @@
+@import '../style.css';
+
+.btn{
+    border: none;
+    margin:0.5rem;
+    box-shadow: rgba(25,25,25,.04) 0 0 1px 0,rgba(0,0,0,.1) 0 3px 4px 0;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.text-decoration-none{
+    text-decoration: none;
+}
+
+.btn-sm{
+    padding:0.5rem;
+    border-radius: 0.5rem;
+    font-size: 1rem;
+}
+
+.btn-primary{
+    background-color: var(--primary-bg);
+    color: var(--white-color); 
+    font-weight: 500;
+}
+
+.btn-primary:hover{
```

Comment:
You can add cursor: pointer on hover on all buttons.

Reply:
added cursor: pointer on hover